tools/min-base-version

```diff
@@ -27,7 +27,7 @@ from glob import glob
 
 proj_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 
-max_version = 0
+max_version = ''
```

Comment:
A bit further on there's a `max_version == 0` check. Won't that always be False now?

Reply:
Well spotted, thanks! Fixed.